### ISSUE TITLE
RUN ISSUE

### ISSUE THREAD

**Issue by JamieK135** [Assistant/ModAssistant#111]
When i try to run ModAssitant.exe this appears even when i run as admin 

![ModAssistant Issue](https://user-images.githubusercontent.com/63051272/78247137-422e8f80-74e2-11ea-9d3b-b08f3b904652.PNG)

**Comment by kauoz** [comment]
I have this issue as well :/

**Comment by Parapass** [comment]
Do you guys have an antivirus running?

**Comment by kauoz** [comment]
> 
> 
> Do you guys have an antivirus running?

Yes, is that what's causing it? It's never been a problem before

**Comment by JamieK135** [comment]
> 
> 
> Do you guys have an antivirus running?
just windows defender i also use ccleaner to get rid of junk

**Comment by Parapass** [comment]
If you guys need help, go to the #pc-help in the [BSMG](https://discord.gg/beatsabermods).

**Comment by Kauoz** [comment]
> 
> 
> > Do you guys have an antivirus running?
> > just windows defender i also use ccleaner to get rid of junk

I've always had ModAssitant.exe on my desktop for easy access but as soon as I moved it to  another location it worked again. It's pretty weird but this fixed it for me.

**Comment by JamieK135** [comment]
> 
> 
> > > Do you guys have an antivirus running?
> > > just windows defender i also use ccleaner to get rid of junk
> 
> I've always had ModAssitant.exe on my desktop for easy access but as soon as I moved it to another location it worked again. It's pretty weird but this fixed it for me.

where did you move it to???

**Comment by Kauoz** [comment]
> 
> 
> > > > Do you guys have an antivirus running?
> > > > just windows defender i also use ccleaner to get rid of junk
> > 
> > 
> > I've always had ModAssitant.exe on my desktop for easy access but as soon as I moved it to another location it worked again. It's pretty weird but this fixed it for me.
> 
> where did you move it to???

I moved it to my games folder which is on my other disk, not my system disk. Don't know if disk location matter but that fixed it

**Comment by JerryFilms** [comment]
I'm having the same issue. I've tried all of the suggested tips but the problem seems to be with BSIPA. As soon as I uninstall it the game works fine but with it the game will not even attempt to start and give the same error message as above. Hopefully we can get a fix on this, I blame the BeatSaber devs for updating their game every 2 minutes.

**Comment by Parapass** [comment]
Once again, go to #pc-help in the BSMG discord.

**Comment by JamieK135** [comment]
> 
> 
> Once again, go to #pc-help in the BSMG discord.

been there no one knows what is going on

**Comment by Parapass** [comment]
Either it can't write or read the config file or your config file is just corrupted.

**Comment by VeikkoLehmuskorpi** [comment]
Simply downloading the latest release and replacing the old ModAssistant.exe file with that fixed it for me.

**Comment by JamieK135** [comment]
> 
> 
> Simply downloading the latest release and replacing the old ModAssistant.exe file with that fixed it for me.

this is the first time installing mod assistant for me

**Comment by megalon** [comment]
Are you still having this issue?
It looks like Windows is trying to access a temp file, which may no longer exist.
On my machine, the file is called `user.config`, but yours is `krxikakp.tmp`

If this error occurs again, can you try to go to the path it provides and see if the file exists?
For example, `C:\Users\<username>\AppData\Local\ModAssistant\ModAssistant.exe_Url_b5ccye5brv14r0rvvr3g5zy3mpn2lvpn\1.1.5.0`

On the technical side, I'm not sure why it would be trying to access a temp file.
Mod Assistant saves settings using `Properties.Settings.Default.Save()`, which should be handled by Windows.
The `Settings` class is auto generated by Visual Studio, and extends the `ApplicationSettingsBase` class <https://docs.microsoft.com/en-us/dotnet/api/system.configuration.applicationsettingsbase?view=netframework-4.6.1>

I can't find any reason why it would be a temp file.
If anyone else has any suggestions here let me know.

**Comment by JerryFilms** [comment]
I was able to fix this issue by filling uninstalling the game. As in going into the .../steamapps/common folder and deleting the entire file (after backing up my downloaded songs and platforms). After redownloading it, installing all the mods, and replacing the downloaded songs it worked perfectly fine.

**Comment by megalon** [comment]
Is anyone still having this issue?

It's been over a year now and I'm wondering if we can close this with the possible solution above.
